code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -1448,6 +1450,19 @@
 		agonyDam += agony_amount * 0.5
 		adjustFireLoss(agonyDam)
 
+/mob/living/simple_animal/emp_act(severity)
+	if(!isSynthetic())
+		return
+	switch(severity)
+		if(1)
+			adjustFireLoss(rand(15, 25))
+		if(2)
+			adjustFireLoss(10, 18)

```

Comment:
I think you're missing a ``rand()`` here?

Reply:
Ah, good catch.